core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -305,16 +305,21 @@ public List<TableIdentifier> listTables(Namespace namespace) {
     }
 
     return fetch(
-        row ->
-            JdbcUtil.stringToTableIdentifier(
```

Comment:
previously this would return the identifier as `.tblName` instead of `tblName` when using an empty namespace